node/clients/search.ts

```diff
@@ -90,83 +85,83 @@ export class Search extends AppClient {
     const pageTypeQuery = !query || query.startsWith('?') ? query : `?${query}`
 
     return this.get<SearchPageTypeResponse>(
-      `/pub/portal/pagetype/${pageTypePath}${pageTypeQuery}`,
-      { metric: 'search-pagetype' }
+      `/api/catalog_system/pub/portal/pagetype/${pageTypePath}${pageTypeQuery}`,
```

Comment:
did you try to call all these functions to see if they still work as expected?

Reply:
No, I navigated on a store, home to PDP, and it worked fine there, but there's probably more cases to test. But the catalog-api-proxy is just a proxy, it doesn't alter anything from the response from the catalog_system. Something that may be different is the authenticatedProxy or something like that, that was a different route on catalog-api-proxy, this dies, but my understanding is that every request already sends the segment and session (if available), and should be authenticated by default. But a good case to test, is the store's that uses this authenticatedProxy, I just don't know which stores are those.